cmd/tasks.go

```diff
@@ -4,34 +4,51 @@ import (
 	"crypto"
 	"fmt"
 
-	"github.com/anchore/syft/syft/artifact"
-
-	"github.com/anchore/syft/syft/sbom"
-
+	"github.com/anchore/syft/internal/config"
 	"github.com/anchore/syft/syft"
+	"github.com/anchore/syft/syft/artifact"
 	"github.com/anchore/syft/syft/file"
+	"github.com/anchore/syft/syft/sbom"
 	"github.com/anchore/syft/syft/source"
 )
 
-type powerUserTask func(*sbom.Artifacts, *source.Source) ([]artifact.Relationship, error)
+type task func(*sbom.Artifacts, *source.Source) ([]artifact.Relationship, error)
 
-func powerUserTasks() ([]powerUserTask, error) {
-	var tasks []powerUserTask
+func tasks(appConfig *config.Application) ([]task, error) {
+	var tasks []task
 
-	generators := []func() (powerUserTask, error){
+	generators := []func() (task, error){
 		catalogPackagesTask,
-		catalogFileMetadataTask,
```

Comment:
given https://github.com/anchore/syft/pull/636/files#r752425018, we could leave all of these and allow configuration to drive task creation